### PR TITLE
Test integral variant for Regge/HHJ

### PR DESCRIPTION
**Pull request by pbrubeck**
# Description

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -83,6 +83,8 @@ jobs:
             --install defcon \
             --install gadopt \
             --install asQ \
+            --package-branch fiat pbrubeck/h1curl \
+            --package-branch finat pbrubeck/hhj \
```
Comment (pbrubeck) [inline .github/workflows/build.yml:87]:
```suggestion
            --package-branch finat pbrubeck/hhj \
            --package-branch tsfc pbrubeck/hhj \
```

File: tests/regression/test_interpolate_cross_mesh.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def parameters(request):
         expected = np.asarray(
             [np.outer(coords[i], coords[i]) for i in range(len(coords))]
         )
-        V_src = FunctionSpace(m_src, "Regge", 2)  # Not point evaluation nodes
+        V_src = FunctionSpace(m_src, "Regge", 2, variant="point")  # Not point evaluation nodes
```
Comment (rckirby) [inline tests/regression/test_interpolate_cross_mesh.py:218]: kwarg and comment seem mismatched?

File: tests/regression/test_tensor_elements.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+def mesh(request):
+    if request.param == "square":
+        return UnitSquareMesh(2, 2)
+    elif request.param == "cube":
+        return UnitCubeMesh(2, 2, 2)
+    else:
+        raise ValueError("Unrecognized mesh type")
+
+
+@pytest.mark.parametrize("family, degree", [
```
Comment (rckirby) [inline tests/regression/test_tensor_elements.py:18]: JM is a tensor element.  Does it want to be tested here as well?

**Review by rckirby — CHANGES_REQUESTED** [review]
Mostly good.  Just a couple of questions.

File: tests/regression/test_tensor_elements.py (hunk 3 above)
Comment (pbrubeck) [reply tests/regression/test_tensor_elements.py:18]: Not really, here we are only testing the distributional tensor-valued elements. JM, AW, and HZ are tested in test_stress_elements.py

File: tests/regression/test_interpolate_cross_mesh.py (hunk 2 above)
Comment (pbrubeck) [reply tests/regression/test_interpolate_cross_mesh.py:218]: The comment was already there. "Not point evaluation nodes" here refers to component point evaluation.

**Review by rckirby — APPROVED** [review]

File: tests/regression/test_interpolate_cross_mesh.py (hunk 2 above)
Comment (rckirby) [reply tests/regression/test_interpolate_cross_mesh.py:218]: Thanks for updating.

**Review by rckirby — APPROVED** [review]

**Pull request merged**